12_listcomprehension_lab/02.py

```diff
@@ -1,6 +1,8 @@
-"""
-Write a python program that takes two words
-as sys.argv and prints only the letters
-common to both
-"""
-
+"""
+Elad Agam - List comprehansion 
+02 - Build all combinations of 3 letter words 
+"""
+
+letters  = [chr(letter) for letter in range(97,123)]
+combinations = [x+y+z for x in letters for y in letters for z in letters]
+print combinations
```

Comment:
I don't understand

On 14 July 2016 at 22:09, Gabor Szabo <notifications@github.com> wrote:

> In 12_listcomprehension_lab/02.py
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/142#discussion_r70865212>
> :
>
> > @@ -1,6 +1,8 @@
> > -"""
> > -Write a python program that takes two words
> > -as sys.argv and prints only the letters
> > -common to both
> > -"""
> > -
> > +"""
> > +Elad Agam - List comprehansion
> > +02 - Build all combinations of 3 letter words
> > +"""
> > +
> > +letters  = [chr(letter) for letter in range(97,123)]
> > +combinations = [x+y+z for x in letters for y in letters for z in letters]
> > +print combinations
>
> 👍
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/142/files/6a9c92faa32484c4e057556af6ff94914394012e..ad737108eee62f9e9e5ed56b2f357128fc220579#r70865212>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARMSUKHXgoIhiNJLOi-PeO0qPA9CbrNaks5qVomBgaJpZM4JMviz>
> .
>